frontend/src/metabase/querying/notebook/utils/steps.ts

```diff
@@ -87,7 +87,18 @@ const STEPS: NotebookStepDef[] = [
   {
     type: "filter",
     clauseType: "filters",
-    valid: query => {
+    valid: (query, stageIndex) => {
+      // do not suggest filtering if there is aggregation without breakout on the last stage
```

Comment:
not last, `previous`

Reply:
fixed e810c883d1fa13de83e310d7a7fab6ae038cfdae